### PR TITLE
Fix displaying custom case images loaded with scout version <= 4.71

### PR DESCRIPTION
**Pull request by northwestwitch**
The images loaded with scout <= 4.71 don't show up. For example on this case -> https://scout-stage.scilifelab.se/cust000/F0006358-mip7

- Check also this test on latest version:  https://github.com/Clinical-Genomics/scout/pull/4125#issuecomment-1768401950

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Deploy this branch on cg-vm1 and make sure that the case above displays the "hello" image on case page

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DNIL
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Main branch:

image


This branch:

image

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -378,7 +378,9 @@ def case(store, institute_obj, case_obj):
 
     if case_obj.get("custom_images"):
         # re-encode images as base64
-        case_obj["custom_images"] = case_obj["custom_images"].get("case_images", {})
+        case_obj["custom_images"] = case_obj["custom_images"].get(
```
Comment (dnil) [inline scout/server/blueprints/cases/controllers.py:381]: 👍

**Review by dnil — APPROVED** [review]
Looks good. That should mean both places the custom images are used now are backwards compatible.

On a side note, maybe we should revisit the versioning scheme interpretation again. These almost-kindof-not-breaking minors are a bit silly. It would be cleaner with just a major to flag a bumpy road ahead on uptake. I could validate a bit more often, and perhaps I could change the wording a bit on the quality documents on the hospital side to not auto-validate on majors but to investigate the nature of the change and validate if needed. 😄

**Comment by dnil** [comment]
> > It would be cleaner with just a major to flag a bumpy road ahead on uptake
> 
> That's my secret dream 😆

Hehe, just remember you were against it last time I tried.. 😂

**Pull request merged**